code/modules/antagonists/heretic/items/forbidden_book.dm

```diff
@@ -39,8 +38,12 @@
 
 	if(book_open)
 		close_animation()
+		RemoveElement(/datum/element/heretic_focus)
+		w_class = WEIGHT_CLASS_SMALL
 	else
 		open_animation()
+		AddElement(/datum/element/heretic_focus)
+		w_class = WEIGHT_CLASS_NORMAL
```

Comment:
Also also: Opening the codex, putting it in your backpack, then taking it back out won't give you heretic focus casting, because the elemnt `on_equip` doesn't have a check for no slot flags

Reply:
how fix

Reply:
you'd have to edit the element to skip the `slot & slotflags` check if the item has no slot, so just `if(item.slot_flags && !(item.slot_flags & slot)`